website/README.md

```diff
@@ -1,22 +1,114 @@
+
+### With Node
+
+If your local development environment has a supported version (v10.0.0+) of [node installed](https://nodejs.org/en/) you can run:
+
+- `npm install`
+- `npm start`
+
+and then visit `http://localhost:3000`.
+
+If you pull down new code from github, you should run `npm install` again. Otherwise, there's no need to re-run `npm install` each time the site is run, you can just run `npm start` to get it going.
+
+## Editing Content
+
+Documentation content is written in [Markdown](https://www.markdownguide.org/cheat-sheet/) and you'll find all files listed under the `/pages` directory.
+
+To create a new page with Markdown, create a file ending in `.mdx` in the `pages/` directory. The path in the pages directory will be the URL route. For example, `pages/hello/world.mdx` will be served from the `/hello/world` URL.
+
+This file can be standard Markdown and also supports [YAML frontmatter](https://middlemanapp.com/basics/frontmatter/). YAML frontmatter is optional, there are defaults for all keys.
+
+```yaml
+---
+title: 'My Title'
+description: "A thorough, yet succinct description of the page's contents"
+---
+
+```
+
+The significant keys in the YAML frontmatter are:
+
+- `title` `(string)` - This is the title of the page that will be set in the HTML title.
+- `description` `(string)` - This is a description of the page that will be set in the HTML description.
+
+> ⚠️Since `api` is a reserved directory within NextJS, all `/api/**` pages are listed under the `/pages/api-docs` path.
+
+### Editing Sidebars
+
+The structure of the sidebars are controlled by files in the [`/data` directory](data).
+
+- Edit [this file](data/docs-navigation.js) to change the **docs** sidebar
+- Edit [this file](data/api-navigation.js) to change the **api docs** sidebar
+
+To nest sidebar items, you'll want to add a new `category` key/value accompanied by the appropriate embedded `content` values.
+
+- `category` values will be **directory names** within the `pages` directory
+- `content` values will be **file names** within their appropriately nested directory.
+
+### Creating New Pages
+
+There is currently a small bug with new page creation - if you create a new page and link it up via subnav data while the server is running, it will report an error saying the page was not found. This can be resolved by restarting the server.
+
+### Changing the Release Version
+
+To change the version of Packer displayed for download on the website, head over to `data/version.js` and change the number there. It's important to note that the version number must match a version that has been released and is live on `releases.hashicorp.com` -- if it does not, the website will be unable to fetch links to the binaries and will not compile. So this version number should be changed _only after a release_.
```

Comment:
Question:

> the website will be unable to fetch links

Does this mean that the binaries are stored 'in the website' ? Or more broadly: why the fetch ?

Reply:
Nope, the binaries are stored in `releases.hashicorp.com` -- we fetch from this API in order to get links to the binaries. For example, for the latest version of packer, we hit this link:

```
https://releases.hashicorp.com/packer/1.5.4/index.json
```

And from the response, can put together the downloads page. This is not actually a change from the way the website previously worked, just wanted extra clarification in the readme 😀

Reply:
( sorry it took so long to reply ) thanks for the explanation; I think it's super nice just like this 🙂. For extra clarity I would may be just precise :

```suggestion
To change the version of Packer displayed for download on the website, head over to `data/version.js` and change the number there. It's important to note that the version number must match a version that has been released and is live on `releases.hashicorp.com` -- if it does not, the website won't be able to compile as the build step validates the existence of the binaries. So this version number should be changed _only after a release_.
```